tracer/test/snapshots/native-profiler-symbols-alpine-arm64.verified.txt

```diff
@@ -0,0 +1,249 @@
+__ctype_get_mb_cur_max
+__cxa_atexit
+__errno_location
+__stack_chk_fail
+__tls_get_addr
+abort
+aligned_alloc
+asprintf
+atexit
+atoi
+atoll
+bcmp
+bind
+blaze_read_elf_build_id
+btowc
+calloc
+ceil
+ceill
+clock_gettime
+close
+closedir
+ddog_crasht_CrashInfo_upload_to_endpoint
```

Comment:
I considered removing all these `ddog_` symbols, as they're all provided by libdatadog, but that would have been a bit of a pain so, meh?